### PR TITLE
update trim.sh script to check packages

### PR DESCRIPTION
**Pull request by pminz**

### CODE REVIEW

File: modules/org/infinispan/dependencies/jdk/openjdk/trim.sh [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
-set -e
+#set -e
 DIR=$(dirname $0)
 PACKAGES=$(cat $DIR/packages.txt)
-rpm -e --nodeps $PACKAGES
+for pkg in $PACKAGES
+do
+    cmd=`rpm --verify $pkg 2> /dev/null`
+    rc=$?
+    if [[ $rc -eq 0 ]]
```
Comment (ryanemerson) [inline modules/org/infinispan/dependencies/jdk/openjdk/trim.sh:9]: There's no need for the `cmd` and `rc` values here, as bash allows a command to be executed directly in the if statement. For example:

```
if rpm --verify $pkg 2> /dev/null; then
  ...
fi
```

File: modules/org/infinispan/dependencies/jdk/openjdk/trim.sh [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+    then
+        echo "${pkg} exists"
+        rpm -e --nodeps $pkg
+    else
+        echo "${pkg} does not exist"
```
Comment (ryanemerson) [inline modules/org/infinispan/dependencies/jdk/openjdk/trim.sh:14]: The `rpm --verify` command already prints the following out to stdout "package %s is not installed", and stdout is not captured by `2> dev/null`. Therefore, we can just remove this else statement and the output will be the following if we have two packages, one which exists and one which does not:

```
%s exists
package %s is not installed
```

File: modules/org/infinispan/dependencies/jdk/openjdk/trim.sh [hunk 3]
```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
-set -e
+#set -e
```
Comment (ryanemerson) [inline modules/org/infinispan/dependencies/jdk/openjdk/trim.sh:2]: We shouldn't uncomment this, as we should handle all error scenarios in the script. If this is removed, then it's a sign the script isn't working as we expect.

**Review by ryanemerson — CHANGES_REQUESTED** [review]

**Comment by pminz** [comment]
@ryanemerson I have made the required changes .

**Comment by ryanemerson** [comment]
Thanks @pminz

**Pull request merged**